### ISSUE TITLE
jx add app jx-app-sso fails with git errors

### ISSUE THREAD

**Issue by tdcox** [jenkins-x/jx#5713]
### Summary 
Running `jx add app jx-app-sso` for a second time results in the following git errors:
```
Set type to githubSet name to GitHubAttempting to cherry pick commits that were on add-app-jx-app-sso-0.0.21 but not yet pushed
WARNING: Unable to cherry-pick ac6b1655557878b1df7c74b899765bd1a268e0bf: git output: error: could not apply ac6b165... Initial import of Dev Environment source
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit': failed to run 'git cherry-pick ac6b1655557878b1df7c74b899765bd1a268e0bf' command in directory '/Users/terry/.jx/environments/dev', output: 'error: could not apply ac6b165... Initial import of Dev Environment source
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit''

error: adding app jx-app-sso version  with alias  using gitops: creating pr for jx-app-sso: pulling environment repo https://github.com/Bootstrap-Jersey/environment-d68-dev.git into /Users/terry/.jx/environments: git output: error: Merging is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.: failed to run 'git merge a9c37b3e-478e-4445-976d-c9f5d9aeb0dd' command in directory '/Users/terry/.jx/environments/dev', output: 'error: Merging is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.'
```

### Jx version

The output of `jx version` is:

```
jx                 2.0.808
Kubernetes cluster v1.13.7-gke.24
kubectl            v1.14.2
helm client        Client: v2.14.0+g05811b8
git                2.21.0
Operating System   Mac OS X 10.13.6 build 17G6030
```

### Jenkins type

- [x] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

**Comment by tdcox** [comment]
This relates to the boot problem with cherry-picking, I believe.

**Comment by tdcox** [comment]
@daveconde I have just tested again and this is worse in jx 2.0.902 because it is now failing the first time that you try to run `jx add app`.

**Comment by patrickleet** [comment]
I'm seeing a slighly different error but also for `jx add app`
```
jx add app replicator
Read credentials for http://chartmuseum.jenkins-x.io from vault helm/repos
Adding missing Helm repo: .. file://../../deploy/helm-chart/kubernetes-replicator
error: : prompting for creds for file://../../deploy/helm-chart/kubernetes-replicator: checking status code of file://../../deploy/helm-chart/kubernetes-replicator/index.yaml: Get file://../../deploy/helm-chart/kubernetes-replicator/index.yaml: unsupported protocol scheme "file"
```

**Comment by ccojocar** [comment]
@tdcox Is this still an issue?

**Comment by deanesmith** [comment]
@tdcox can you confirm the recent fix?  Please close the issue if you're satisfied.

**Comment by abayer** [comment]
Yeah, I'm 99.999% sure I fixed this when I fixed #6350.
